Configuration/AlCa/python/autoCond.py

```diff
@@ -66,7 +66,7 @@
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
     'phase1_2018_cosmics_peak'     : '121X_upgrade2018cosmics_realistic_peak_v4',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2021
-    'phase1_2021_design'           : '121X_mcRun3_2021_design_v7',
+    'phase1_2021_design'           : '121X_mcRun3_2021_design_v8',
```

Comment:
Why not 121X_mcRun3_2021_realistic_v9? 

IIUC, Geometry record used in autoCondModifiers in this PR, XMLFILE_Geometry_121DD4hepV1_Extended2021_mc should compare with XMLFILE_Geometry_121YV1_Extended2021_mc which comes with v9,
https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/121X_mcRun3_2021_realistic_v8/121X_mcRun3_2021_realistic_v9

Reply:
Hi @srimanob we are using `121X_mcRun3_2021_realistic_v9`, please see line
https://github.com/cms-sw/cmssw/blob/da7a5c044593e2187fadcc5f00a273560fabd95d/Configuration/AlCa/python/autoCond.py#L71
what I changed here is the design GT, by adding a record that was forgotten a bit earlier.

Reply:
Ah, I looks the wrong line, sorry.